labs/docker-101/2.md

```diff
@@ -36,15 +36,15 @@ CMD ["/hello"]
 
 Here's what it tells `docker build` to do:
 
-`FROM scratch`
+* `FROM scratch`
```

Comment:
I'd remove the bullets - just in case it would cause confusion about what goes into the `Dockerfile` line-by-line.